src/components/map/map.tsx

```diff
@@ -21,6 +21,7 @@ export type MapProps = {
   onSelectStopPlace?: (id: string) => void;
   mapLegs?: MapLegType[];
   initialZoom?: number;
+  bounds?: [[number, number], [number, number]];
 };
```

Comment:
All of these props are optional now, how can I know which ones cancel others and what not? I also support Mikael's thought in the comment regarding moving bounds.

Reply:
We could do something like this to indicate that `position` and `initialZoom` are redundant when `mapLegs` is defined, and check for what props are given, but I'm unsure if this is easier to understand. We also need to include that none is required somehow. Do you think moving the comment from line 52 is enough? 

```ts

export type MapProps = {
  layer?: string;
  onSelectStopPlace?: (id: string) => void;
} & (
  | {
      position?: Position;
      initialZoom?: number;
    }
  | {
      mapLegs?: MapLegType[];
    }
);

```

Reply:
Is this more accurate ?

```
export type MapProps = {
  layer?: string;
  onSelectStopPlace?: (id: string) => void;
} & (
  | {
      position: Position;
      initialZoom: number;
    }
  | {
      mapLegs: MapLegType[];
    }
  | {}
);
```

Reply:
Yes, that is more accurate 👍 I'll update